apps/src/music/blockly/MusicBlocklyWorkspace.ts

```diff
@@ -180,6 +175,16 @@ export default class MusicBlocklyWorkspace {
     this.workspace = null;
   }
 
+  /**
+   * Hide any custom fields that are showing.
+   */
+  hideChaff() {
+    if (this.headlessMode) {
```

Comment:
Should we also confirm `this.workspace` is defined?

Reply:
I wasn't able to find a case where it wasn't, but certainly doesn't hurt to check.  Added!